test/test_quantized.py

```diff
@@ -417,6 +417,93 @@ def test_qlinear(self, batch_size, input_channels, output_channels, use_bias, us
         self.assertEqual(Y_fp32, Y_fp32_ref,
                          message="torch.ops.quantized.fbgemm_linear_dynamic results are off")
 
+    @given(
+        batch_size=st.integers(1, 4),
+        input_channels=st.integers(16, 32),
+        output_channels=st.integers(4, 8),
+    )
+    def test_qlinear_legacy(self, batch_size, input_channels, output_channels):
```

Comment:
Is this a dup? I see the same code under test_qlinear in the same file in the other diff, please update

Reply:
@raghuramank100 : This is for the unit test of the original `torch.fbgemm_linear_quantize_weight`, `torch.fbgemm_pack_quantized_matrix`, and `torch.fbgemm_linear_int8_weight` APIs for `torch.jit.quantized` workflow.